tests/console/docker_image_rpm.pm

```diff
@@ -24,32 +25,21 @@ sub run {
     my $image_name;
     my $image_path;
 
+    install_docker_when_needed();
+
     if (is_tumbleweed) {
         $image_name = 'opensuse/tumbleweed:current';
         $image_path = '/usr/share/suse-docker-images/native/*-image*.tar.xz';
 
         # For Tumbleweed, the image is wrapped inside an RPM
-        zypper_call "in docker opensuse-tumbleweed-image";
-    }
-    elsif (is_leap('15.0+')) {
-        $image_name = "opensuse-leap-$version:current";
-        $image_path = "/usr/share/suse-docker-images/native/opensuse-leap${version}-image.tar.xz";
-
-        # For Leap, the docker image is the ISO asset
-        my $image_filename = get_required_var('ISO');
-        $image_filename =~ s/^.*\///;
-        my $image_url = autoinst_url("/assets/other/$image_filename");
-        assert_script_run "curl $image_url --create-dirs -o $image_path";
+        zypper_call "ar -fG https://download.opensuse.org/repositories/Virtualization:containers:images:openSUSE-Tumbleweed/container/Virtualization:containers:images:openSUSE-Tumbleweed.repo";
+        zypper_call "in opensuse-tumbleweed-image";
+        zypper_call "rr https://download.opensuse.org/repositories/Virtualization:containers:images:openSUSE-Tumbleweed/container/Virtualization:containers:images:openSUSE-Tumbleweed.repo";
```

Comment:
As we have same repo here and at line #35, what's about putting it in a variable?

Reply:
It is done now.